aiobungie/internal/factory.py

```diff
@@ -389,7 +389,9 @@ def _set_character_attrs(self, payload: typedefs.JSONObject) -> character.Charac
             class_type=enums.Class(payload["classType"]),
             emblem=assets.Image(payload.get("emblemBackgroundPath")),
             emblem_icon=assets.Image(payload.get("emblemPath")),
-            emblem_hash=int(payload["emblemHash"]),
+            emblem_hash=int(payload["emblemHash"])
```

Comment:
You're calling `.get` twice.
```suggestion
            emblem_hash=int(payload["emblemHash"]) if "emblemHash" in payload else None
```